.github/actions/todb-init/Dockerfile

```diff
@@ -15,9 +15,11 @@
 # specific language governing permissions and limitations
 # under the License.
 
-FROM golang:1.14.2-alpine
+FROM golang:1.14.2-buster
 
-RUN apk add --no-cache git gcc gettext postgresql-client
+RUN apt-get update &&\
+	apt-get install -y --no-install-recommends git gcc gettext-base postgresql-client-11 &&\
```

Comment:
This builds for me on alpine if the `musl-dev` package is added (16MB).

Reply:
dang, that's huge. But probably substantially less than the difference between buster and alpine.